dagster/terraform/gcp/main.tf

```diff
@@ -26,25 +26,6 @@ module "gcs_buckets" {
   project_id            = var.project_id
   bucket_names          = [var.dagster_bucket]
   service_account_email = module.dagster-workload-identity.gcp_service_account_email
+  location              = var.bucket_location
```

Comment:
that might remove previously created cluster (in US by default), do we consider it as breaking change? cc @michaeljguarino

Reply:
it definitely could replace buckets in place, I suppose we just need to do it though, so let's just push it